nakadi-java-client/src/main/java/nakadi/Problem.java

```diff
@@ -145,6 +145,24 @@ public static Problem observerProblem(String title, String detail) {
         .type(OBSERVER_TYPE);
   }
 
+  /**
+   * Quick way to create a Problem object that indicates a server responded with an error but the message could not be converted to a Problem
+   * object automatically; typically this happens when a server responds with a non-json payload.
+   *
+   * @param statusCode the HTTP status code of the server's response
+   * @param body the HTTP body of the server's response
+   * @param errorDetail details about why it failed to convert the server's response to a Problem object
+   * @return a Problem object with a status of {@param statusCode} and a type of "about:contract_retryable"
+   */
+  public static Problem rawProblem(int statusCode, String body, String errorDetail) {
+    return new Problem()
+            .title(body)
```

Comment:
We might want to use a substring here in case the body is too long?

Reply:
sure, what do you think would be a reasonable safe-guard?

Reply:
Maybe 200 characters?
(At least the "title" name lets me think of something short-ish.)

Reply:
done